.envrc

```diff
@@ -0,0 +1,14 @@
+path="./.venv"
```

Comment:
I think a more descriptive name like "venv path" is going to make the rest of the file more clear to the reader. Also, a note on convention, we usually write constants in `CAPS_SNAKE_CASE`.